src/Components/NavBar/NavBar.tsx

```diff
@@ -50,6 +50,7 @@ export const NavBar: React.FC = track(
   const { xs, sm } = useMedia()
   const isMobile = xs || sm
   const isLoggedIn = Boolean(user)
+  const getNotificationCount = () => cookie.get("notification-count") || 0
```

Comment:
Wondering why defer this?

Reply:
For the client; `cookie` requires a window, so putting it behind a function serves as a guard.

Reply:
Ah right, sorry- I've asked this before! Thanks, makes perfect sense.